backend/internal/server/apis/stats/penalty.go

```diff
@@ -9,15 +9,29 @@ import (
 	"github.com/jak103/powerplay/internal/server/apis"
 	"github.com/jak103/powerplay/internal/server/services/auth"
 	"github.com/jak103/powerplay/internal/utils/locals"
+	"github.com/jak103/powerplay/internal/utils/log"
 	"github.com/jak103/powerplay/internal/utils/responder"
 )
 
 func init() {
-	//apis.RegisterHandler(fiber.MethodGet, "/penaltyTypes", auth.Public, getPenaltyTypes)
+	apis.RegisterHandler(fiber.MethodGet, "/penaltyTypes", auth.Public, getPenaltyTypes)
 	apis.RegisterHandler(fiber.MethodGet, "/penalties", auth.Public, getPenaltiesHandler)
 	apis.RegisterHandler(fiber.MethodPost, "/penalties", auth.Public, postPenaltyHandler)
 }
 
+func getPenaltyTypes(c *fiber.Ctx) error {
+
+	log.Debug("Handling getting all penalty types")
```

Comment:
We need to get the logger from the locals first in order to get logging to work
`log := locals.Logger(c)`

However, Dr. Christensen said we don't need this first log statement, as the logger automatically does this.

Reply:
Ok I got rid of the first log statement. I didn't need to initialize it with locals though to get it to print since I'm using the public functions in the log package. Maybe I need this explained more to me though. I was seeing duplicate debugs for that log that was removed so I can verify it logged, but maybe the way I was doing it is incorrect

Reply:
Got it, I need to understand this better as well